yoti-sdk-impl/src/test/java/com/yoti/api/client/spi/remote/SimpleProfileTest.java

```diff
@@ -139,15 +154,57 @@ public void findAttributeStartingWith_shouldReturnNullForMismatchingType() {
 
     @Test
     public void findAttributeStartingWith_shouldReturnValueForMatchingKey() {
-        Profile profile = new SimpleProfile(map(STARTS_WITH + ":restOfKey", INTEGER_VALUE));
+        Profile profile = new SimpleProfile(asList(STARTS_WITH + ":restOfKey", INTEGER_VALUE));
 
         Integer result = profile.findAttributeStartingWith(STARTS_WITH, Integer.class);
 
         assertEquals(INTEGER_VALUE, result);
     }
 
-    private Map<String, Object> map(String key, Object o) {
-        return singletonMap(key, o);
+    @Test
+    public void getAttributeSourcesShouldIncludeDrivingLicence() throws ParseException, IOException {
+        com.yoti.api.client.spi.remote.proto.AttrProto.Attribute attribute = buildAnchoredAttribute(GIVEN_NAMES_ATTRIBUTE, "A_Given_NAME", TestAnchors.DL_ANCHOR);
```

Comment:
I think we'd be ok with __AttrProto.Attribute__

Reply:
:+1: